dev/test-dependencies.sh

```diff
@@ -34,8 +34,8 @@ HADOOP_MODULE_PROFILES="-Phive-thriftserver -Pmesos -Pkubernetes -Pyarn -Phive \
     -Pspark-ganglia-lgpl -Pkinesis-asl -Phadoop-cloud"
 MVN="build/mvn"
 HADOOP_HIVE_PROFILES=(
-    hadoop-2.7-hive-2.3
-    hadoop-3.2-hive-2.3
+    hadoop-2-hive-2.3
```

Comment:
unrelated but the `hive-2.3` seems redundant, since we don't support alternative Hive profile right now